### PR TITLE
feat(MembershipsSdkAdapter): return the sharing status of each member

### PR DESCRIPTION
**Pull request by lungancatalin**
Update SDK Adapter to send a sharing status for a member

### CODE REVIEW

File: src/MembershipsSDKAdapter.test.js [hunk 1]
```diff
@@ -32,10 +32,12 @@ describe('Memberships SDK Adapter', () => {
                 {
                   id: 'id',
                   muted: false,
+                  sharing: false,
                 },
                 {
                   id: 'mutedPerson',
                   muted: true,
+                  sharing: true,
```
Comment (lalli-flores) [inline src/MembershipsSDKAdapter.test.js:40]: It's funny that the muted person is sharing 😂

**Review by lalli-flores — COMMENTED** [review]
Seems like you need to update your branch. There are conflicts with master

File: src/MembershipsSDKAdapter.test.js (hunk 1 above)
Comment (lungancatalin) [reply src/MembershipsSDKAdapter.test.js]: Yes, it's funny, but it could be a real case :))

**Review by lalli-flores — APPROVED** [review]

**Pull request merged**